packages/typespec-bump-deps/src/cli/cli.ts

```diff
@@ -23,7 +23,12 @@ async function main() {
   console.log("The following is a mapping between packages and the versions we will update them to");
   // eslint-disable-next-line no-console
   console.log(packageToVersionRecord);
-  const packageJsonPaths = process.argv.slice(2);
+  const args = process.argv.slice(2);
+  const addRushOverridesArg = "--add-rush-overrides";
+  const addNpmOverridesArg = "--add-npm-overrides";
+  const addRushOverrides = args.includes(addRushOverridesArg);
+  const addNpmOverrides = args.includes(addNpmOverridesArg);
+  const packageJsonPaths = args.filter((arg) => arg !== addRushOverridesArg && arg !== addNpmOverridesArg);
```

Comment:
I probably should use yargs, but maybe I do it in another PR.

Reply:
There is actually a built-in parser for args in node now:
https://nodejs.org/api/util.html#utilparseargsconfig

It is simple but that will cover those use cases very well I think.

Reply:
Changed to parseArgs.

This TS error on type get me puzzled for quite some time.

TS won't accept
```ts
  const options = {
    "add-rush-overrides": {
      type: "boolean",
    },
    "add-npm-overrides": {
      type: "boolean",
    },
  };
```

So I had to explicitly write have the union there. Not sure if there is better way to write this.
```ts
  const booleanTypeStr: "string" | "boolean" = "boolean";
  const options = {
    "add-rush-overrides": {
      type: booleanTypeStr,
    },
    "add-npm-overrides": {
      type: booleanTypeStr,
    },
  };
```

Reply:
hhm, weird, let me try out

Reply:
ok just had to use `as const` so typscript keeps the exact type and doesn't downgrade to `string`

```
const options = {
    "add-rush-overrides": {
      type: "boolean",
    },
    "add-npm-overrides": {
      type: "boolean",
    },
  } as const;
```